README.md

```diff
@@ -1,178 +1,176 @@
 
-(To build TimescaleDB from source, see instructions in [_Building from source_](https://github.com/timescale/timescaledb/blob/main/docs/BuildSource.md).)
+# Install TimescaleDB
 
-### Using TimescaleDB
-
-TimescaleDB scales PostgreSQL for time-series data via automatic
-partitioning across time and space (partitioning key), yet retains
-the standard PostgreSQL interface.
+Installation options are:
 
-In other words, TimescaleDB exposes what look like regular tables, but
-are actually only an
-abstraction (or a virtual view) of many individual tables comprising the
-actual data. This single-table view, which we call a
-[hypertable](https://tsdb.co/GitHubTimescaleHypertable),
-is comprised of many chunks, which are created by partitioning
-the hypertable's data in either one or two dimensions: by a time
-interval, and by an (optional) "partition key" such as
-device id, location, user id, etc.
+- Install the [platform-specific package](https://docs.timescale.com/self-hosted/latest/install/).
+- [Build from source](https://docs.timescale.com/self-hosted/latest/install/installation-source/).
 
-Virtually all user interactions with TimescaleDB are with
-hypertables. Creating tables and indexes, altering tables, inserting
-data, selecting data, etc., can (and should) all be executed on the
-hypertable.
+We recommend not using TimescaleDB with PostgreSQL 17.1, 16.5, 15.9, 14.14, 13.17, 12.21.
+These minor versions [introduced a breaking binary interface change][postgres-breaking-change] that,
+once identified, was reverted in subsequent minor PostgreSQL versions 17.2, 16.6, 15.10, 14.15, 13.18, and 12.22.
+When you build from source, best practice is to build with PostgreSQL 17.2, 16.6, etc and higher.
+Users of [Timescale Cloud](https://console.cloud.timescale.com/) and Platform packages built and
+distributed by Timescale are unaffected.
 
-From the perspective of both use and management, TimescaleDB just
-looks and feels like PostgreSQL, and can be managed and queried as
-such.
+TimescaleDB comes in the following editions: Apache 2 and Community. See the [documentation](https://docs.timescale.com/about/latest/timescaledb-editions/) for differences between them.
 
-#### Before you start
+For reference and clarity, all code files in this repository reference [licensing](https://github.com/timescale/timescaledb/blob/main/tsl/LICENSE-TIMESCALE) in their header. Apache-2 licensed binaries can be built by passing `-DAPACHE_ONLY=1` to `bootstrap`.
 
 PostgreSQL's out-of-the-box settings are typically too conservative for modern
-servers and TimescaleDB. You should make sure your `postgresql.conf`
-settings are tuned, either by using [timescaledb-tune](https://github.com/timescale/timescaledb-tune)
+servers and TimescaleDB. Make sure your `postgresql.conf`
+settings are tuned, by either using [timescaledb-tune](https://github.com/timescale/timescaledb-tune)
 or doing it manually.
 
-#### Creating a hypertable
+# Create a hypertable
```

Comment:
```suggestion
## Create a hypertable
```

Please use a `##`, a `#` translates to a `<h1>` and only of those should be present for SEO purposes.